components/admin_console/system_users/system_users_dropdown/system_users_dropdown.tsx

```diff
@@ -194,10 +194,19 @@ export default class SystemUsersDropdown extends React.PureComponent<Props, Stat
                     messageForUsersWithBotAccounts = (
                         <FormattedMarkdownMessage
                             id='deactivate_member_modal.desc.for_users_with_bot_accounts'
-                            defaultMessage='This action deactivates {username}.\n \n * They will be logged out and not have access to any teams or channels on this system.\n * Bot accounts they manage will be disabled along with their integrations. To enable them again, go to [Integrations > Bot Accounts]({siteURL}/_redirect/integrations/bots). [Learn more about bot accounts](!https://mattermost.com/pl/default-bot-accounts).\n \n \n'
+                            defaultMessage='This action deactivates {username}.\n \n * They will be logged out and not have access to any teams or channels on this system.\n * Bot accounts they manage will be disabled along with their integrations. To enable them again, go to [Integrations > Bot Accounts]({siteURL}/_redirect/integrations/bots). <link>Learn more about bot accounts</link>.\n \n \n'
```

Comment:
This one had Markdown AND another link in here, and the Markdown can't just be replaced by a custom `<strong>` function since this message is a paragraph followed by a list. I think we'll have to split this into a few different strings and use HTML for the formatting instead of Markdown

Reply:
Hi @hmhealey,
I'm having an issue trying to fix this. As the strings belong to a variable and I get an error when I try adding something to the variable (it can only have one element). Can you show me the way?

Reply:
I'm not sure what you mean exactly, but if you mean that you're trying to add multiple `FormattedMessage`s here where there used to only one one and TypeScript is complaining about that, you can wrap it in a [`Fragment`](https://reactjs.org/docs/fragments.html). Using the shorthand for Fragments, it would look like
```js
messageForUsersWithBotAccounts = (
    <>
        <p><FormattedMessage/></p>
        <ul>
            <li><FormattedMessage/></li>
        </ul>
    </>
);
```

Hopefully that answers your question